Bottom: 3537b1cdf3b21530a3bbff7fc68e2d0c6a8c246d
Top:    24f3b65b8b73e54c7b72dd8a0a6d2b9f83c0f072
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-03 13:47:42 +0000

Fix build warnings in gtkhid-gl.c


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index f9dd1da..e782ae7 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -395,6 +395,7 @@ ghid_draw_grid (BoxType *drawn_area)
   glEnable (GL_STENCIL_TEST);
 }
 
+#if 0
 /* XXX: Refactor this into hidgl common routines */
 static void
 load_texture_from_png (char *filename)
@@ -435,6 +436,7 @@ load_texture_from_png (char *filename)
 
   g_object_unref (pixbuf);
 }
+#endif
 
 static void
 ghid_draw_bg_image (void)
@@ -1668,8 +1670,8 @@ fill_board_outline_holes (hidGC gc, const BoxType *drawn_area)
 static void
 GhidDrawMask (int side, BoxType * screen)
 {
-  static bool first_run = true;
-  static GLuint texture;
+//  static bool first_run = true;
+//  static GLuint texture;
   int thin = TEST_FLAG(THINDRAWFLAG, PCB) || TEST_FLAG(THINDRAWPOLYFLAG, PCB);
   LayerType *Layer = LAYER_PTR (side == TOP_SIDE ? top_soldermask_layer : bottom_soldermask_layer);
   struct poly_info info;
@@ -1762,7 +1764,7 @@ GhidDrawMask (int side, BoxType * screen)
 
   gui->graphics->use_mask (HID_MASK_OFF);
 
-  first_run = false;
+//  first_run = false;
 }
 
 static void
